src/content/en/updates/2017/07/devtools-release-notes.md

```diff
@@ -0,0 +1,394 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New features and changes coming to DevTools in Chrome 61.
+
+{# wf_updated_on: 2017-07-10 #}
+{# wf_published_on: 2017-07-10 #}
+{# wf_tags: chrome61,devtools #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: New features and changes coming to DevTools in Chrome 61. #}
```

Comment:
Instead of using a stock phrase every time, why don't you tease some of the features? I would especially recommend this since your stock phrase is little more than a repeat of the title.